extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -246,7 +246,8 @@
         ['set', 'dimension16', getKruxSegment()],                              // Krux Segment
         ['set', 'dimension17', window.wgWikiVertical],                         // Vertical
         ['set', 'dimension18', window.wgWikiCategories.join(',')],             // Categories
-        ['set', 'dimension19', window.wgArticleType]                          // ArticleType
+        ['set', 'dimension19', window.wgArticleType],                          // ArticleType
+        ['set', 'dimension20', window.wgABPerformanceTest || 'not set']        // Performance A/B testing
```

Comment:
What should be the default value here? Three approaches are used currently - `undefined`, `''`, `'not set'`.